packages/sample-app/sample_app/openai_agents_realtime_example.py

```
@@ -0,0 +1,684 @@
+    print("\n" + "=" * 60)
+    print("Handoff demo completed!")
+    print("OpenTelemetry spans captured handoff events between agents")
+    print("=" * 60)
+
+
+def check_requirements():
+    """Check if required packages are installed."""
+    missing = []
+
+    try:
+        import agents  # noqa: F401
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)